gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -101,6 +102,30 @@ const changeUnissuedFacilityPreview = async (req, res) => {
   }
 };
 
+// when changing unissued facility from application preview page
+// renders about-facility change page for unissued facilities
+const changeIssuedToUnissuedFacility = async (req, res) => {
+  const { params, query } = req;
+  const { dealId, facilityId } = params;
+  let { facilityType } = query;
+  facilityType = facilityType || FACILITY_TYPE.CASH;
+  const facilityTypeString = FACILITY_TYPE[facilityType?.toUpperCase()].toLowerCase();
```

Comment:
Should this be a helper function?

Reply:
this is exactly how it is currently for facilities currently.  Only difference is that I have taken out the status field

Reply:
Lets chat about to it on Monday, after stand up.

Reply:
done

Reply:
@Zainzzkk  I can't see the changes

Reply:
I still can't see the changes for this @Zainzzkk  :)